bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -410,13 +412,14 @@ private void create_22_BlogHeader(final StringBuilder sb) {
 
    private void create_24_Tour(final StringBuilder sb) {
 
-      String tourTitle = _tourData.getTourTitle();
       String tourDescription = _tourData.getTourDescription();
-      String tourWeather = WeatherUtils.buildWeatherDataString(_tourData, true, true, true);
+      final boolean isDescription = StringUtils.hasContent(tourDescription);
 
-      final boolean isDescription = tourDescription.length() > 0;
-      final boolean isTitle = tourTitle.length() > 0;
-      final boolean isWeather = tourWeather.length() > 0;
+      String tourTitle = _tourData.getTourTitle();
+      final boolean isTitle = StringUtils.hasContent(tourTitle);
+
+      String tourWeather = WeatherUtils.buildWeatherDataString(_tourData, true, true, true);
+      final boolean isWeather = StringUtils.hasContent(tourWeather);
```

Comment:
The old code was better readable where the `is...` vars are all together

![image](https://user-images.githubusercontent.com/1283445/220578327-64a1d920-473c-4420-a5d1-be2c0c46fe30.png)

Reply:
I tried that because I found it more readable to have each category together (example: `tourDescription` with `isDescription`) but I reverted that.

Reply:
My code formatting was similar like in the `enableControls` methods, first collect all `is...` var's and then apply them, e.g.

![image](https://user-images.githubusercontent.com/1283445/220836385-0a579141-9b69-487c-976d-26edf3328694.png)

Sometimes the `is...` vars depend on each of them and then it is better readable for me when they are all together

Reply:
I will follow this pattern